### DISCUSSION TITLE
Entity linking with workable exemples

### DISCUSSION THREAD

**Discussion by igormorgado** [explosion/spaCy#12989] [Help: Coding & Implementations]
I'm reading spaCy documentation and I'm having issues with Entity linking shown at 
[[official documentaion](https://spacy.io/usage/linguistic-features#entity-linking)](https://spacy.io/usage/linguistic-features#entity-linking), the code does not run simply because there isn't a "my_custom_el_pipeline". This seems an issue in documentation since the examples do not work.

Here I'm asking two things.

First, how this work? How can I write a simple code that shows entity linking?

Second, is possible to fix the documentation with runnable examples?

**Comment by danieldk** [comment]
Thanks for reporting this! `my_custom_el_pipeline` would be a spaCy pipeline that you trained or saved yourself. But this is not really clear from the documentation. I'll convert this to an issue, since we should make this clearer in the example.

**Comment by svlandeg** [comment]
Hi @igormorgado,

I've submitted a PR to update the documentation: https://github.com/explosion/spaCy/pull/13071

I also wanted to refer you to this tutorial that shows how to build a knowledge base & how to train the entity linker: https://github.com/explosion/projects/tree/v3/tutorials/nel_emerson. Working through that code will hopefully clarify how to work with entity linking in spaCy - but let us know if you have further questions.
